src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -102,9 +55,127 @@ public HelpWindow() {
     }
 
     private void setHelpText() {
-        Text text = new Text(HELP_TEXT);
-        text.getStyleClass().add("help-text");
-        helpTextFlow.getChildren().add(text);
+        helpTextFlow.getChildren().addAll(
+                createHeader("BA€ 1.0 Help                                                    "),
+                createParagraph(
+                        "Welcome to BA€ 1.0, your powerful ally in optimizing recurring sales.",
+                        "This guide will help you navigate the key features and commands of BA€."
+                ),
+                createHeader("Key Features and Commands"),
+                createFeature(
```

Comment:
Might want to eventually retrieve/reference these help text(s) for Commands from the existing static constants in the respective Command implementations. That way there is always only a single source-of-truth for the help text(s) to avoid discrepancies in future iterations and changes.